apps/competitions-api/src/routes/competitions/getCompetitionByEid.ts

```diff
@@ -0,0 +1,43 @@
+import { Router } from 'express';
+import { prisma } from '@competition-manager/prisma';
+import { z } from 'zod';
+import { parseRequest } from '@competition-manager/utils';
+import { Competition$ } from '@competition-manager/schemas';
+
+export const router = Router();
+
+const Params$ = z.object({
+    eid: z.string()
+});
+
+router.get(
+    '/:eid',
+    parseRequest('params', Params$),
+    async (req, res) => {
+        const { eid } = Params$.parse(req.params);
+        const competition = await prisma.competition.findUnique({
+            where: {
+                eid,
+                publish: true
+            },
+            select: {
```

Comment:
On peut pas avoir un select all ou un exclude si on ne doit pas tout prendre

Reply:
pas compris

Reply:
Au lieu de select chaque champs comme tu as fait ce serait mieux de tout sélectionné par défaut et de exclure les champs qu'on ne veut pas 
(Toujours la même idée si tu rajoute un champs dans competition normalement tu veux aussi l'avoir ici mais ça se fait sans rajouter du code si tu fais avec excluse)

Reply:
Prisma does not natively provide an omit option in its query API to directly exclude fields. The select property is designed to explicitly include fields you want, rather than exclude fields. 
allez hop dans ta gueule